src/components/common/Actions.vue

```diff
@@ -224,8 +224,9 @@ export default class Actions extends Mixins(AmalgamationMixin, CommonMixin,
         // Re-fetch the business table data. We do this in case one of the businesses has changed (eg,
         // became historical) while the draft was open.
         const holdingPrimary = await this.refetchAmalgamatingBusinessesInfo()
+
         // If there's a holding or primary business, re-fetch its data and update the prepopulated data.
-        // This will overwrite office addresses, directors and share structure.
+        // This will overwrite office addresses, directors, share structure, legal name and legal type.
         if (holdingPrimary) await this.updatePrepopulatedData(holdingPrimary)
```

Comment:
This is called when the user submits an amalgamation filing. This is to catch the scenario where the holding/primary business name or type has changed in the background while this amalgamation remained open.